src/jimgw/cli/_transforms.py

```
@@ -0,0 +1,468 @@
+        # GaussianPrior: use the inverse CDF (probit) to map u ∈ (0,1) → x ∈ ℝ.
+        # GaussianTransform forward maps u → mu + sigma*ndtri(u); reverse maps x → ndtr((x-mu)/sigma) ∈ (0,1).
+        return [
+            reverse_bijective_transform(
+                GaussianTransform(
+                    name_mapping=([f"{name}_unit"], [name]),
+                    mu=spec.loc,
+                    sigma=spec.scale,
+                )
+            )
+        ]
+    assert False, f"Unknown prior spec type for '{name}': {type(spec)}"
```

Ruff (0.15.12)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)